infra/core/compute/ubuntu-jumpbox.bicep

```diff
@@ -46,19 +28,13 @@ param name string
 @description('The name of the linux PC.  By default, this will be automatically constructed by the resource name.')
```

Comment:
nit: consistency in naming - we're calling it a Virtual Machine elsewhere, not PC